internal/pkg/diff/reporter.go

```diff
@@ -24,11 +37,24 @@ func (r *Reporter) Report(rs cmp.Result) {
 		if len(pathStr) > 0 {
 			r.diffs = append(r.diffs, fmt.Sprintf("  \"%s\":", pathStr))
 		}
+
+		// Format relations diff
+		if r.relationsDiff(vx, vy) {
```

Comment:
`Reporter` formatuje vysledky, ... `relations` si musime naformatovat rucne, ... kedze diff porovnava slices tak, ze kontroluje poradie ... a aj ak by sa obe `Relations` zoradili, ... tak ak niekde chyba nejaka polozka, tak to uz dava zle vysledky, vzdy porovnava rovnake indexy v slice.